Pipeliner/ContentView.swift

```diff
@@ -28,7 +28,7 @@ struct ContentView: View {
 
     
     private func isFormValid() -> Bool {
-        if projectId.isEmpty {
+        if self.selection != .GITHUB && projectId.isEmpty {
```

Comment:
You probably should check the .self.selection is .GITHUB or .GITLAB.

Reply:
@Jakub-Vacek This is similar to the check on the [line 65](https://github.com/DXHeroes/Pipeliner/pull/24/files/e56148f3e5249bca0af396bcd4a241f0a89d65c8#diff-877d640ca70419ba4a17f2a2098b29c280d4fb011ac5985147f04bba9217c56cR65): `projectId` makes sense for GitLab only.
If you prefer, I can update it to emphasize GitLab
```suggestion
        if self.selection == . GITLAB && projectId.isEmpty {
```

Reply:
Oh, okey I see that now. `!= .GITHUB` is fine. Maybe u cen add comment saying something like "GitHub service does not use projectId"

Reply:
Done